rasterio/warp.py

```diff
@@ -12,11 +13,13 @@
 from rasterio._warp import (
     _transform_geom, _reproject, _calculate_default_transform)
 from rasterio.enums import Resampling
-from rasterio.env import ensure_env, Env
+from rasterio.env import ensure_env
 from rasterio.transform import guard_transform
 
 
-def transform(src_crs, dst_crs, xs, ys, zs=None, **options):
+@ensure_env
+def transform(src_crs, dst_crs, xs, ys, zs=None):
```

Comment:
@sgillies FYI the `**options` argument was added in `1.0a` (https://github.com/mapbox/rasterio/commit/eee5c9d481f0c10cc01e52fbb9981f43a3d3a16b), is undocumented, and is untested, so I'm inclined to just remove rather than deprecate.